<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::TabPosition::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::TabShape::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Laskenta</string>
         </attribute>
         <attribute name="toolTip">
          <string>Laske uudelleen</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QTableView" name="laskentaTableView"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="LaskeButton">
            <property name="text">
             <string>Laske</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Materiaalit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Historia</string>
         </attribute>
         <attribute name="toolTip">
          <string>Katso vanhoja laskelmia</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_3">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="2">
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="Lataalasku">
            <property name="text">
             <string>Lataa lasku?</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTableWidget" name="tableWidget_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox_2">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Vanhat laskut</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Uusin lasku</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Arvot</string>
         </attribute>
         <attribute name="toolTip">
          <string>Muuta arvoja laskuille</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="8" column="2">
           <widget class="QPushButton" name="arvotdeleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="arvotAddNewButton">
            <property name="text">
             <string>Add new</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Materiaali:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" rowspan="8" colspan="2">
           <widget class="QTableView" name="arvotTableView"/>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QLineEdit" name="arvotMateriaaliLineEdit">
            <property name="minimumSize">
             <size>
              <width>282</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QLineEdit" name="arvothintaLineEdit">
            <property name="minimumSize">
             <size>
              <width>276</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Hinta:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QPushButton" name="arvotUpdateButton">
            <property name="text">
             <string>Update/Save changes</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="arvotComboBox"/>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="arvotUpdateTable">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="view-refresh"/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Tietojen muokkaus</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>id:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="arvotIdLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
